Navidux/.swiftpm/xcode/package.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you should add workspace in `.gitignore`

Reply:
Added this file to gitignore